src/components/record/related-table-actions.tsx

```diff
@@ -764,11 +769,13 @@ const RelatedTableActions = ({
             }
           >
             <a
-            className={`chaise-btn bulk-edit-link ${
-              tertiary ? 'chaise-btn-tertiary dropdown-button' : 'chaise-btn-secondary'
-            }`}
+              className={`chaise-btn bulk-edit-link
```

Comment:
Do we want to change the tooltip when the button is disabled (Similar to how we're changing the "Add"/"link" button tooltips when they are disabled?